README.md

```diff
@@ -84,3 +84,34 @@ key:
 
 This configuration must be made at the root package level (the package
 _consuming_ configuration providing packages).
+
+### Behavior with the whitelisted require-dev dependencies
+
+In a Laminas MVC application, when you have a whitelisted package declared as "require-dev", the module will add its config to the `config/development.config.php` file if it exists.
+
+```json
+"require-dev": {
+  "laminas/laminas-validator": "*"
+},
+"extra": {
+  "laminas": {
+    "component-whitelist": [
+      "laminas/laminas-validator"
+    ]
+  }
+}
+```
+
+```php
+<?php
+// config/development.config.php
+
+return [
+    // Additional modules to include when in development mode
+    'modules' => [
+        'Laminas\Validator', // automatically added line. 
+    ],
+];
+```
+
+> Using composer 2, the dependencies of the require-dev package will install in `modules.config.php` file if there are whitelisted.
```

Comment:
The readme file is the wrong place. Please add this section to the documentation. The source can be found in the `docs` directory. Thanks in advance!

(The whole readme file is wrong here because the content is duplicated and duplicated content always ends in missing updates on one side.)

Reply:
Thank you very much for your guidance:-), I moved the new section to the same section at `docs/index.md`.